### ISSUE TITLE
Cool things about GitHub I learned

### ISSUE THREAD

**Issue by noorka** [MaryIsbell/ShipboardTheatricals#34]
So, I have had a lot of programming assignments in the past few weeks that my Dad has been helping me with. Throughout the process he complained about all the zip files I sent him and said it would be so much easier as a git. He didn't realize I have a github, so I made a repository for my code and my Dad taught me how to integrate it on my machine. 

So now I have this thing called SourceTree that handles all of the commits I make to the repository and can show me the changes I have made to the document before I commit. I think it might be similar to the github for desktop we discussed last time. I really like this format because I can see what I've done, but if I pull files off of the git I can also see the changes others have made to the files as compared to the copies I have on my personal machine. 

Long story short, I would like to use this for the ShipboardTheatricals repository as well. I could branch, but I think it would work the best if I had the git clone url for the repository to add to my machine repositories. Would you be comfortable with giving me that URL, or would you prefer I continue to commit changes in our typical workflow?

**Comment by MaryIsbell** [comment]
Great! Let me know if you need anything from me to do this.
